test/extensions/filters/listener/tls_inspector/tls_inspector_test.cc

```diff
@@ -360,51 +445,30 @@ TEST_P(TlsInspectorTest, NotSsl) {
 
   // Use 100 bytes of zeroes. This is not valid as a ClientHello.
   data.resize(100);
-
+#ifdef WIN32
+  EXPECT_CALL(os_sys_calls_, readv(_, _, _))
```

Comment:
Can you add a utility function somewhere that makes the proper EXPECT for win32/non-win32? It will cut down a lot of the copy/paste in these tests.

Reply:
will do.